src/audio/module_adapter/module/volume/volume.c

```diff
@@ -267,15 +267,18 @@ static inline int32_t volume_windows_fade_ramp(struct vol_data *cd, int32_t ramp
  */
 
 /* Note: Using inline saves 0.4 MCPS */
-static inline void volume_ramp(struct vol_data *cd)
+static inline void volume_ramp(struct processing_module *mod)
```

Comment:
The checkpatch fail is from commit message typo "If tha gain".

Reply:
thanks, I am waiting the CI test result. And then push again.